src/packages/cli/src/cli.ts

```diff
@@ -20,7 +21,7 @@ const logAndForceExit = (messages: any[], exitCode = 0) => {
     (process.stdout as any)._handle.setBlocking(true);
   }
   try {
-    messages.forEach(console.log);
+    messages.forEach(message => console.log(message));
```

Comment:
This was logging the message, the index, and the array for each message in `messages`. I fixed that so it only logs the message now.